tfjs-backend-webgpu/src/kernels/clip_webgpu.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+import {util} from '@tensorflow/tfjs-core';
+import {computeDispatch, flatDispatchLayout} from '../webgpu_util';
+
+import {WebGPUProgram} from './webgpu_program';
+
+export class ClipProgram implements WebGPUProgram {
+  outputShape: number[];
+  userCode: string;
+  variableNames = ['A'];
+  dispatchLayout: {x: number[]};
+  dispatch: [number, number, number];
+  workGroupSize: [number, number, number] = [64, 1, 1];
+
+  constructor(outputShape: number[], minVal: number, maxVal: number) {
+    this.outputShape = outputShape;
+    const size = util.sizeFromShape(this.outputShape);
+    this.dispatchLayout = flatDispatchLayout(this.outputShape);
+    this.dispatch = computeDispatch(
```

Comment:
@qjia7 Have you tried profiling clip with different `workGroupSize` values? What about increasing the `workPerThread` (defaults to `1` if not set)? 

You could look at `pad_webgpu` for an example of `workPerThread` > 1. Curious how these parameters potentially tradeoff with each other in terms of performance.